main.gms

```diff
@@ -601,7 +601,7 @@ $setGlobal cm_nash_mode  parallel      !! def = parallel
 $SetGlobal cm_quick_mode  off          !! def = off
 $setGLobal cm_debug_preloop  off !! def = off
 $setGlobal c_EARLYRETIRE       on         !! def = on
-$setGlobal cm_OILRETIRE  on        !! def = on
+$setGlobal cm_OILRETIRE  off        !! def = on
```

Comment:
I think main.gms changes to switch settings not specific to this change shouldn't be committed?

Reply:
Yes, Robert is right. this needs to be reverted